plugins/obs-webrtc/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+project(obs-webrtc)
+
+include(FetchContent)
+
+FetchContent_Declare(
+  Corrosion
+  GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
+  GIT_TAG v0.3.1)
+
+FetchContent_MakeAvailable(Corrosion)
```

Comment:
Test if corrosion is already available system-wide or in the prefix before using `FetchContent_MakeAvailable`.

Add corrosion as a module for the Flatpak manifest, the tooling should be available before reaching `"obs"` module.

Reply:
Will do today!

Reply:
Fixed!